python/lsst/ip/diffim/dipoleFitTask.py

```diff
@@ -102,10 +107,16 @@ def setDefaults(self):
                               "base_PixelFlags",
                               "base_SkyCoord",
                               "base_PsfFlux",
+                              "base_GaussianCentroid",
+                              #"base_GaussianFlux",
```

Comment:
My bad. I completely forgot to remove the commented-out debug code before pushing. Thanks for noting it -- I'll remove all of the comments that you've identified.